library/src/main/res/xml/provider_paths.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<paths xmlns:android="http://schemas.android.com/apk/res/android">
```

Comment:
`android` is a redundant namespace declaration here.

Reply:
updated